holmes/plugins/toolsets/__init__.py

```
@@ -121,9 +121,17 @@ def load_python_toolsets(
     ]
 
     if not DISABLE_PROMETHEUS_TOOLSET:
-        from holmes.plugins.toolsets.prometheus.prometheus import PrometheusToolset
-
-        toolsets.append(PrometheusToolset())
+        try:
+            from holmes.plugins.toolsets.prometheus.prometheus import PrometheusToolset
+
+            toolsets.append(PrometheusToolset())
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)